ci-operator/step-registry/aws/c2s/instance-metadata/aws-c2s-instance-metadata-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: aws-c2s-instance-metadata
+  from: upi-installer
```

Comment:
Same (also `upi-installer` is not part of the payload, so it may fail)

Reply:
> also upi-installer is not part of the payload, so it may fail

Sorry I do not quite understand this part, could you please explain more about this? 
The reason I use `upi-installer` is just for `jq` and `yo-go` tools, they are available in `upi-installer` image.

Reply:
I guess `from: upi-installer` rely on job definition, so for `aws-c2s-instance-metadata` step, we can use `centos8` here, should be enough for it.